chemprop/v2/data/utils.py

```diff
@@ -2,65 +2,253 @@
 from typing import Sequence
 
 import numpy as np
-from rdkit.Chem.Scaffolds import MurckoScaffold
 
 
 from chemprop.v2.data.datapoints import MoleculeDatapoint
+from chemprop.v2.data.datasets import MoleculeDataset
+
+from astartes import train_val_test_split, train_test_split
+from astartes.molecules import train_val_test_split_molecules, train_test_split_molecules
+
+import copy
+from logging import Logger
+import pickle
+from random import Random
+from typing import Tuple
+import os
+
+from chemprop.data import log_scaffold_stats
+from chemprop.args import TrainArgs
```

Comment:
These two imports are v1 code. We will have to use the v2 equivalence if there's any.

Reply:
import of trainargs has been removed, log import has been fixed